src/native.js

```diff
@@ -230,9 +356,16 @@ export function getAllAssetsMessage(data, adObject) {
   const message = {
     message: 'assetResponse',
     adId: data.adId,
-    assets: []
   };
 
+  if (adObject.native.ortb) {
+    Object.keys(adObject.native).forEach(key => {
+      message[key] = adObject.native[key];
```

Comment:
Is this right? From my reading of the PUC PR it looks like it expects these under `message.ortb` ?

Reply:
Actually disregard this, of course `ortb` is also being copied over.